# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../musl/pkg/ebtables/ebtables-use-standard-int-types.patch
# Copyright (C) 2014 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

From b94ddfd6ea097db2ee8414556094ed37e8b20611 Mon Sep 17 00:00:00 2001
From: Christian Wiese <chris@opensde.org>
Date: Mon, 28 Apr 2014 00:09:25 +0200
Subject: changed to use standard int types

---
 extensions/ebt_ip6.c   |    4 ++--
 extensions/ebt_limit.c |   10 +++++-----
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/extensions/ebt_ip6.c b/extensions/ebt_ip6.c
index 0465e77..a1fc1ae 100644
--- a/extensions/ebt_ip6.c
+++ b/extensions/ebt_ip6.c
@@ -53,8 +53,8 @@ static const struct option opts[] =
 
 struct icmpv6_names {
 	const char *name;
-	u_int8_t type;
-	u_int8_t code_min, code_max;
+	uint8_t type;
+	uint8_t code_min, code_max;
 };
 
 static const struct icmpv6_names icmpv6_codes[] = {
diff --git a/extensions/ebt_limit.c b/extensions/ebt_limit.c
index ee40e5c..d189a09 100644
--- a/extensions/ebt_limit.c
+++ b/extensions/ebt_limit.c
@@ -59,11 +59,11 @@ static void print_help(void)
 "                                default %u\n", EBT_LIMIT_BURST);
 }
 
-static int parse_rate(const char *rate, u_int32_t *val)
+static int parse_rate(const char *rate, uint32_t *val)
 {
 	const char *delim;
-	u_int32_t r;
-	u_int32_t mult = 1;  /* Seconds by default. */
+	uint32_t r;
+	uint32_t mult = 1;  /* Seconds by default. */
 
 	delim = strchr(rate, '/');
 	if (delim) {
@@ -151,7 +151,7 @@ static void final_check(const struct ebt_u_entry *entry,
 struct rates
 {
 	const char *name;
-	u_int32_t mult;
+	uint32_t mult;
 };
 
 static struct rates g_rates[] =
@@ -162,7 +162,7 @@ static struct rates g_rates[] =
 	{ "sec", EBT_LIMIT_SCALE }
 };
 
-static void print_rate(u_int32_t period)
+static void print_rate(uint32_t period)
 {
 	unsigned int i;
 
-- 
1.7.2.3

